Makefile

```diff
@@ -191,7 +191,7 @@ gen-manifests: manifest-templator
 		./hack/generate-manifests.sh
 
 gen-k8s: $(CONTROLLER_GEN) $(apis_sources)
-	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
+	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./pkg/...;./cmd/...;./test/...;./tools/..."
```

Comment:
can we somehow exclude the problematic ones instead ?
if not, those are all we need ? (it means that in case we will need more in the future we should remember to add them, but i doubt it will happen)